pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
Your suggestion worked for me and I've included it in https://github.com/NixOS/nixpkgs/pull/21775/commits/5bb4d2b7320c05c1c1df5651084741d92b4d5053, @LnL7. I'd be happy to squash these two commits if y'all prefer.